src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -36,10 +36,11 @@ code-component .toolbar .carbon-taskbar {
 }
 
 code-component .toolbar .carbon-taskbar.monaco-toolbar .monaco-action-bar .codicon.action-label {
-	height: 20px;
-	padding-bottom: 10px;
+	height: 25px;
```

Comment:
Where did 25px come from?

Reply:
I made the width and height slightly bigger so that the highlight can be seen. If I keep them at 20px, the highlight can't really be noticed.